mongodb/scripts/courses.csv

```diff
@@ -0,0 +1,44 @@
+Course,Name,Offered,Units
+CSC 111,Fundamentals of Programming with Engineering Applications,"[F, Sp]",1.5
+ENGR 110,Design and Communication I,[F],2.5
+ENGR 130,Introduction to Professional Practice,"[F, Sp]",0.5
+MATH 100,Calculus I,"[F, Sp, Su]",1.5
+MATH 109,Introduction to Calculus,"[F, Sp, Su]",1.5
+MATH 110,Matrix Algebra for Engineers,[F],1.5
+PHYS 110,Introductory Physics I,"[F, Sp]",1.5
+CSC 115,Fundamentals of Programming II,"[F, Sp, Su]",1.5
+ENGR 120,Design and Communication II,[Sp],2.5
+ENGR 141,Engineering Mechanics,"[Sp, Su]",1.5
+MATH 101,Calculus II,"[F, Sp, Su]",1.5
+PHYS 111,Introductory Physics II,"[Sp, Su]",1.5
+ECE 255,Introduction to Computer Architecture,[F],1.5
+CSC 230,Introduction to Computer Architecture,"[F, Sp, Su]",1.5
+CHEM 101,Fundamentals of Chemistry from Atoms to Materials,"[F, Su]",1.5
+ECE 260,Continuous-Time Signals and Systems,"[F, Su]",1.5
+MATH 122,Logic and Foundations,"[F, Sp, Su]",1.5
+SENG 265,Software Development Methods,"[F, Sp, Su]",1.5
+STAT 260,Introduction to Probability and Statistics I,"[F, Sp, Su]",1.5
+CSC 225,Algorithms and Data Structures I,"[F. Sp, Su]",1.5
+ECE 310,Digital Signal Processing I,"[Sp, Su]",1.5
+ECON 180 ,Introduction to Economics and Financial Project Evaluation,"[F, Sp, Su]",1.5
+SENG 275,Software Testing,"[Sp, Su]",1.5
+SENG 310,Human Computer Interaction,"[F, Sp, Su]",1.5
+ECE 363,Communication Networks,[Sp],1.5
+CSC 361,Computer Communications and Networks,"[F, Sp]",1.5
+CSC 226,Algorithms and Data Structures II,"[F, Sp, Su]",1.5
+ECE 360,Control Theory and Systems I,"[F, Sp]",1.5
+SENG 321,Requirements Engineering,"[F, Sp]",1.5
+SENG 371,Software Evolution,[Sp],1.5
+ECE 355,Microprocessor-Based Systems,[F],1.5
+CSC 355,Digital Logic and Computer Organization,[F],1.5
+CSC 320,Foundations of Computer Science,"[F, Sp, Su]",1.5
+CSC 360,Operating Systems,"[F, Sp, Su]",1.5
+CSC 370,Database Systems,"[F, Sp, Su]",1.5
+SENG 350,Software Architecture and Design,[F],1.5
+SENG 360,Security Engineering,[F],1.5
+SENG 426,Software Quality Engineering,[Su],1.5
+SENG 440,Embedded Systems,[Su],1.5
+SENG 499,Design Project II,[Su],1.5

```

Comment:
Do we need to store credits for the courses? We might need to add a credit field to courses struct in that case.

Reply:
I believe all classes are just 1.5 credits but perhaps I am incorrect? As far as preqs go I don't have that data on hand but I can certainly collect it. For now it may just be worth making a small ticket in the future. @jberryman566 how would you like to proceed regarding preqs? I mean it is kinda important data to have

Reply:
was something that was not provided. All the data I collected was just transferring from their docs. Sadly, we will need to comb through uvic's schedules to figure out the prereqs. As far as credits go, they are not super important, I just used them as they were provided. If the other company is not using them we can disregard. Questions like these should be brought up in Mondays meetings with the other company.

Reply:
@jakecomp Sounds good. Everything else looks good!